### ISSUE TITLE
SF1Edit 4.1.5f x64 unable to load Starfields base game esm's

### ISSUE THREAD

**Issue by MyersP** [TES5Edit/TES5Edit#1356]
**What is the version of xEdit you are using?**
SF1Edit 4.1.5f x64 (C0DCE642)

Starfield Game version v1.12.30

**Describe the bug**
Loading errors on certain record types.
Just load the base game esm.

**To reproduce**
1. Open the Starfield.esm
2. Note the loading errors flowing from the loading

**Expected behavior**
I was expecting to load my esm's in an error free way (on official esm's)

1st Try: 
       SF1Edit crashed while I was waiting for it to load.
2nd Try:
       SF1Edit killed my computer, it was on, but no video found and non responsive. Did a hard reboot.
A day later I tried again:
       SF1Edit used over 50GB of memory (I have 64) but it finished loading.
       That was while I was writing this report up.

**Additional context**
<details><summary>Example of log message:</summary>
<p>
SF1Edit 4.1.5f x64 (C0DCE642) starting session 2024-06-11 18:06:07
The Source Code Form is available at https://github.com/TES5Edit/TES5Edit

Using Starfield Data Path: B:\SteamLibrary\steamapps\common\Starfield\Data\
Using Backup Path: B:\SteamLibrary\steamapps\common\Starfield\Data\SF1Edit Backups\
Using Scripts Path: B:\StarfieldMods\SF1Edit 4.1.5f\Edit Scripts\
Using Cache Path: B:\SteamLibrary\steamapps\common\Starfield\Data\SF1Edit Cache\
Using ini: B:\SteamLibrary\steamapps\common\Starfield\\Starfield.ini
Using custom ini: C:\Users\user\Documents\My Games\Starfield\StarfieldCustom.ini
Using save path: C:\Users\user\Documents\My Games\Starfield\__MO_Saves\
Using plugin list: C:\Users\user\AppData\Local\Starfield\Plugins.txt
Using settings file: C:\Users\user\AppData\Local\Starfield\Plugins.sf1viewsettings
Using language: En
Using general string encoding: 1252  (ANSI - Latin I)
Using translatable string encoding: 1252  (ANSI - Latin I)
Using VMAD string encoding: 65001 (UTF-8)
Loading active plugin list: C:\Users\user\AppData\Local\Starfield\Plugins.txt

[00:26] Background Loader: [Starfield.esm] Start loading or building reference info.
[00:26] Background Loader: [Starfield.exe] Start loading or building reference info.
[00:26] Background Loader: [BlueprintShips-Starfield.esm] Start loading or building reference info.
[00:26] Background Loader: [SFBGS008.esm] Start loading or building reference info.
[00:26] Background Loader: [SFBGS007.esm] Start loading or building reference info.
[00:26] Background Loader: [SFBGS006.esm] Start loading or building reference info.
[00:26] Background Loader: [Starfield.exe] Done building reference info.
[00:26] Background Loader: [SFBGS003.esm] Start loading or building reference info.
[00:26] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord MOD2 32444F4D
[00:26] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord BFCE 45434642
[00:26] Background Loader: [SFBGS008_En.DLSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS008_En.ILSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS008_En.STRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: Errors were found in: CityNewAtlantisWell "The Well" [CELL:000F4ED4]
[00:26] Background Loader: Contained subrecords: EDID BFCB(<Unknown: BGSCityMapsUsage_Component>) MOD2 BFCE FULL DATA XCLL LTMP XCLW XILS XCIM XLCN XCMO XOWN XCAS XXXX XTV2 XEMP
[00:26] Background Loader: [SFBGS003_En.DLSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord MOD2 32444F4D
[00:26] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord BFCE 45434642
[00:26] Background Loader: [SFBGS003_En.ILSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS003_En.STRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: Error: record RACE contains unexpected (or out of order) subrecord SADD 44444153
[00:26] Background Loader: Errors were found in: SFBGS003_HumanRace "Human" [RACE:0300009D]
[00:26] Background Loader: Contained subrecords: 
... Looked like every type of record multiple times
[00:26] Background Loader: [SFBGS006_En.DLSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS006_En.ILSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS006_En.STRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS007_En.DLSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS007_En.ILSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [SFBGS007_En.STRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: Error: record FLST contains unexpected (or out of order) subrecord ANAM 4D414E41
[00:26] Background Loader: Errors were found in: GPO_PEO_GameplayOptionDifficultySliders [FLST:FE002047]
[00:26] Background Loader: Contained subrecords: EDID LNAM LNAM LNAM LNAM ANAM
[00:26] Background Loader: Errors were found in: CityCydoniaMainLevel "Cydonia - Central Hub" [CELL:002B3DA2]
[00:26] Background Loader: Contained subrecords: EDID BFCB(<Unknown: BGSCityMapsUsage_Component>) MOD2 BFCE FULL DATA XCLL LTMP XCLW XILS XCIM XLCN XCMO XCGD XCAS TODD XXXX XTV2 XEMP
[00:26] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord MOD2 32444F4D
[00:26] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord BFCE 45434642
[00:26] Background Loader: [Starfield_En.DLSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:26] Background Loader: [Starfield_En.ILSTRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:27] Background Loader: [Starfield_En.STRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:27] Background Loader: Errors were found in: CityNeonCore "Neon Core" [CELL:00071D5A]
[00:27] Background Loader: [SFBGS007.esm] Done building reference info.
[00:27] Background Loader: Contained subrecords: EDID BFCB(<Unknown: BGSCityMapsUsage_Component>) MOD2 BFCE FULL DATA XCLL LTMP XCLW XILS XLCN XCMO XCAS XCIM XEMP XXXX XTV2
[00:27] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord MOD2 32444F4D
[00:27] Background Loader: Error: record CELL contains unexpected (or out of order) subrecord BFCE 45434642
[00:27] Background Loader: [USU_En.STRINGS] Using encoding (from language): 1252  (ANSI - Latin I)
[00:27] Background Loader: Errors were found in: CityCydoniaMainLevel02 "Cydonia - Residential" [CELL:0004A181]
[00:27] Background Loader: Contained subrecords: EDID BFCB(<Unknown: BGSCityMapsUsage_Component>) MOD2 BFCE FULL DATA XCLL LTMP XCLW XILS XCIM XLCN XCMO XCGD TODD XTV2 XEMP
... Many more like those
[00:27] Background Loader: Error: record LCTN contains unexpected (or out of order) subrecord ACUN 4E554341
[00:27] Background Loader: Error: record LCTN contains unexpected (or out of order) subrecord ACSR 52534341
[00:27] Background Loader: Error: record LCTN contains unexpected (or out of order) subrecord ACID 44494341
[00:27] Background Loader: Error: record LCTN contains unexpected (or out of order) subrecord ACEP 50454341
[00:27] Background Loader: Errors were found in: CityAkilaCityLocation [LCTN:00010DFB]
... Many more like those

[00:36] Background Loader: Errors were found in: SFBGS006AddToList_SMSSet_HabVariants_Deimos_2L2W1H [FLST:02000341]
[00:36] Background Loader: Contained subrecords: EDID LNAM LNAM LNAM LNAM LNAM LNAM ANAM
.. Many more like those

</p>
</details>

**Comment by jbostrus** [comment]
Yeah the F build is not supporting the latest update. There is a new G build with all the stuff updated that came with the release of the Creation Kit 2 days ago. You will need to join the Discord server and visit the `#xedit-builds` channel to get the 4.1.5g build.

**Comment by Jaheay** [comment]
> Yeah the F build is not supporting the latest update. There is a new G build with all the stuff updated that came with the release of the Creation Kit 2 days ago. You will need to join the Discord server and visit the `#xedit-builds` channel to get the 4.1.5g build.

I can't find a download link for the g version on the releases page anymore :/
